docs/content/ingestion/firehose.md

```diff
@@ -119,10 +118,9 @@ The below configurations can be optionally used for tuning the Firehose performa
 
 ### IngestSegmentFirehose
 
-This Firehose can be used to read the data from existing druid segments.
-It can be used to ingest existing druid segments using a new schema and change the name, dimensions, metrics, rollup, etc. of the segment.
+This Firehose can be used to read the data from existing druid segments, potentially using a new schema and changing the name, dimensions, metrics, rollup, etc. of the segment.
 This Firehose is _splittable_ and can be used by [native parallel index tasks](./native_tasks.html#parallel-index-task).
-A sample ingest Firehose spec is shown below:
+This firehose uses `map` typed parsers. A sample ingest Firehose spec is shown below:
```

Comment:
Does it? The `ingestSegment` firehose example specs in the docs use `string` type parsers.

Reply:
I uh.. maybe? I referenced the ingest segment firehose unit tests which, which is using map parser with time and dims parsespec. I'll have another look and maybe change those other docs if that is not the case.

Reply:
So, this works with any type of parser, either wrapping it in a `TransformingInputRowParser` or `TransformingStringInputRowParser` depending on the parser type specified, and strictly using it to create a transform spec